Experimental/tests/Ranges/ImmutableRangeListTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+using NUnit.Framework;
+
+namespace CodeJam.Ranges
+{
+    [TestFixture]
+    public class ImmutableRangeListTests
+    {
+        [Test]
+        public void Contains()
+        {
+			// TODO:
+        }
+
+
+        [Test]
+        public void Invert()
+        {
+            var list = Range.Create(20, 30).ToRangeList().Add(Range.Create(11, 12, false, true));
```

Comment:
This Create(.., .., false, true) syntax looks ugly. Please, consider using enum values like Include/NotInclude instead of bools.